src/platform/plugins/shared/chart_expressions/expression_legacy_metric/public/components/metric_value.test.tsx

```diff
@@ -49,6 +50,7 @@ describe('MetricVisValue', () => {
         labelConfig={labelConfig}
       />
     );
+
     expect(component.find('button').exists()).toBe(true);
```

Comment:
just add `data-test-subj="metric_value_button"` in source

```tsx
  if (onFilter) {
    return (
      <button
        data-test-subj="metric_value_button"
        css={{ display: 'block' }}
        onClick={() => onFilter()}
        title={i18n.translate('expressionLegacyMetricVis.filterTitle', {
          defaultMessage: 'Click to filter by field',
        })}
      >
        {metricComponent}
      </button>
    );
  }
```

and here

```suggestion
    expect(component.find('[data-test-subj="metric_value_button"]').exists()).toBe(true);
```

Reply:
done